samples/project-data/project-data-set-sound.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2018-present, Renderforest, LLC.
+ * All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory.
+ */
+
+const RenderforestClient = require('../../lib/client')
+
+const Renderforest = new RenderforestClient({ signKey: '<signKey>', clientId: -1 })
+
+Renderforest.getProjectData(6221499)
+  .then((projectDataInstance) => {
+    return projectDataInstance.setSounds([
+      {
```

Comment:
it's the same as the old one, please check template fonts example in project-data